tests/acceptance/features/webUISharingInternalUsersBlacklisted/shareWithUsers.feature

```diff
@@ -1,4 +1,5 @@
-@issue-ocis-1317
+@issue-ocis-1317 @skipOnOCIS
+# exclude-groups-from-sharing configuration is not available now in the oCIS webUI
```

Comment:
I didn't mention the issue link in comment because at the top @issue-ocis-1317 tag is present which is related to the issue